java/client/test/org/openqa/selenium/support/ui/FluentWaitTest.java

```diff
@@ -172,7 +172,8 @@ public void timeoutMessageIncludesLastIgnoredException() {
   @Test
   public void timeoutMessageIncludesCustomMessage() {
     TimeoutException expected = new TimeoutException(
-      "Timed out after 0 seconds: Expected custom timeout message");
+        "Could not match predicate: Expected custom timeout message "
```

Comment:
I think many users will be confused by the term 'predicate', since it's an internal detail of how expected conditions are constructed.

Can we change that bit back to be "Timed out, %s (tried for %d ..."